.github/workflows/build-nightly-ci-docker-images.yml

```diff
@@ -15,16 +15,6 @@ jobs:
     name: "Nightly PyTorch + Stable TensorFlow"
     runs-on: [intel-cpu, 8-cpu, ci]
     steps:
-      - name: Cleanup disk
-        run: |
-          sudo ls -l /usr/local/lib/
-          sudo ls -l /usr/share/
-          sudo du -sh /usr/local/lib/
-          sudo du -sh /usr/share/
-          sudo rm -rf /usr/local/lib/android
-          sudo rm -rf /usr/share/dotnet
-          sudo du -sh /usr/local/lib/
-          sudo du -sh /usr/share/
```

Comment:
we don't need this anymore after using `[intel-cpu, 8-cpu, ci]`. I forgot to remove this in #31119